core/ts/MaciState.ts

```diff
@@ -5,15 +5,12 @@ import { Poll } from "./Poll";
 import { TreeDepths, MaxValues } from "./utils/utils";
 import { STATE_TREE_ARITY, BlankStateLeaf, BlankStateLeafHash } from "./utils/constants";
 
-/*
- * File Overview:
- * - IMaciState: Interface detailing `MaciState`'s public API.
- * - MaciState: Core class implementing the above interface.
+/**
+ * Represents the public API of the MaciState class.
  */
-
 interface IMaciState {
```

Comment:
Let's use consistent new lines. I'd suggest to remove all the newlines between interface props unless you'd like to add description for props.